ndk-build/src/manifest.rs

```diff
@@ -258,6 +262,39 @@ pub struct Permission {
     pub max_sdk_version: Option<u32>,
 }
 
+/// Android [package element](https://developer.android.com/guide/topics/manifest/queries-element#package).
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct Package {
+    #[serde(rename(serialize = "android:name"))]
+    pub name: String,
+}
+
+/// Android [provider element](https://developer.android.com/guide/topics/manifest/queries-element#provider).
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct QueryProvider {
+    #[serde(rename(serialize = "android:authorities"))]
+    pub authorities: String,
+
+    // The specs say only `authorities` attribute is the only one needed for providers contained in `queries`
+    // however aapt throws errors requireing a `android:name` attribute, nd-build should be using appt2?
```

Comment:
To quote the spec:

> Note: There are some restrictions on the options that you can include in this [<provider>](https://developer.android.com/guide/topics/manifest/provider-element) element, compared to a typical <provider> manifest element. Usually, you only specify the android:authorities attribute.

Terribly sad that they don't cover _what_ restrictions are in place on top of a normal `<provider>` element :-1: 

Can you please fix all the typos in this comment? While at it, we still use `aapt`, not `aapt2`.

Reply:
> While at it, we still use `aapt`, not `aapt2`.

Yes that is what i meant, i think the issue is because of using aapt instead o aapt2 but that is just my guess.

Reply:
You do need to test that and be sure that `aapt` vs `aapt2` is indeed the cause?